images/generate-assets.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-# These assets are used almost exclusively for IE8 fallback, so don't need retina versions
```

Comment:
Hidden in a commit message: `SCALE=2 ./generate-assets.sh`, `SCALE=3 ./generate-assets.sh`.